CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [Unreleased]
+
+### Added
+
+- Added `--http_api_max_payload_size` parameter to configure the maximum payload
+  size for PUT and PATCH requests.
```

Comment:
Nit: Even if this was backported, now should we add it under `0.25.1`. Or will this be done in the release commit ?

Reply:
This will be done in the release commit, using `prepare-release` devtool command. I just wanted to make sure everything looks good before continuing with that step.